deploy_guides/MacOS_TFLite_Guide.md

```diff
@@ -0,0 +1,60 @@
+# How to Run TensorFlow Lite Models on MacOS
+This guide shows how to set up a TensorFlow Lite Runtime environment on a MacOS device. We'll use [Anaconda](https://www.anaconda.com/) to create a Python environment to install the TFLite Runtime in. It's easy!
+
+## Step 1. Download and Install Anaconda
+First, install [Anaconda](https://www.anaconda.com/), which is a Python environment manager that greatly simplifies Python package management and deployment. Anaconda allows you to create Python virtual environments on your PC without interfering with existing installations of Python. Go to the [Anaconda Downloads page](https://www.anaconda.com/products/distribution) and click the Download button.
+
+When the download finishes, open the downloaded .exe file and step through the installation wizard. Use the default install options.
+
+## Step 2. Set Up Virtual Environment and Directory
+First open up the terminal by opening a Finder window, and press 'Command + Shift + U', and then select Terminal. We'll create a folder called `tflite1` directly in the Home folder (under your username) - you can use any other folder location you like, just make sure to modify the commands below to use the correct file paths. Create the folder and move into it by issuing the following commands in the terminal:
+
+```
+mkdir tflite1
```

Comment:
```suggestion
mkdir ~/tflite1
```

Reply:
I know that opening a terminal should put you in the Home directory, but I still like specifying the full filepath to where users should create the folder (in this case, `~/tflite1`). Otherwise, inexperienced can get confused about where the folder should be created.